libs/pages/settings/src/lib/ui/page-organization-credentials/page-organization-credentials.tsx

```diff
@@ -0,0 +1,222 @@
+import { CloudProviderEnum, type ClusterCredentials, type CredentialCluster } from 'qovery-typescript-axios'
+import { useMemo } from 'react'
+import { useParams } from 'react-router-dom'
+import { match } from 'ts-pattern'
+import { useDeleteCloudProviderCredential } from '@qovery/domains/cloud-providers/feature'
+import { ClusterAvatar, ClusterCredentialsModal, CredentialsListClustersModal } from '@qovery/domains/clusters/feature'
+import { useOrganizationCredentials } from '@qovery/domains/organizations/feature'
+import { Button, Icon, Indicator, Tooltip, useModal, useModalConfirmation } from '@qovery/shared/ui'
+
+const toCloudProvider = (cloudProvider: ClusterCredentials['object_type']): CloudProviderEnum => {
```

Comment:
Perhaps the name isn't particularly relevant; something like `convertToCloudProviderEnum` might be more suitable, wdyt?